src/NServiceBus.Core/Diagnostics/NServiceBusDiagnosticsInfo.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace NServiceBus;
+
+static class NServiceBusDiagnosticsInfo
+{
+    //TODO should we use a "default" namespace to allow easier separation with verbose/advanced namespaces when using wildcards? (e.g. NServiceBus.*, NServiceBus.Default.*, NServiceBus.Verbose.*)
```

Comment:
I think our current version of the guidelines states that it should be named after the nuget package, that means they are automatically "NServiceBus" namespace and then it's more consistent with the package (because the transport/persistence package namespaces are highly inconsistent). The only downside with this approach is that once we add "optional" spans it's not easy to opt-out when you use `AddSource("NServiceBus.*")` since there is no exclude-API

Reply:
I think what we have might be OK for now. Our documentation for each component will include instructions for how to enable diagnostic tracing for it. Adding a source for `NServiceBus.*` might be considered an advanced scenario.

Reply:
Currently, the instrumentation name is shared between meters and activities. Is that okay? Do we see that potentially diverging in the future? In which namespace should the downstream meters (if any) should be placed? I think this is similar to the question/proposal Laila is raising in the above comment.